primap2/csg/_strategies/gaps.py

```diff
@@ -12,23 +14,16 @@ class Gap:
 
     Attributes
     ----------
-    type :
+    type
```

Comment:
Seems that my pycharm setting for docstrings is not correct, that added the colons, I think